deployer/src/deployer/upload.py

```diff
@@ -61,6 +64,15 @@ def count(self, task):
             self.uploaded_files += 1
             self.uploaded_files_size += task.size
 
+    def delete(self, task):
+        if task.skipped:
+            self.skipped += 1
+        elif task.error:
+            self.failed += 1
+        else:
+            self.deleted_files += 1
+            # self.deleted_files_size += task.size
```

Comment:
Here's a suggestion. I think we should remove this method, and instead do the following:
- add an `is_deletion = False` class attribute to the `UploadTask` base class below
- add an `is_deletion = True` class attribute to the `DeleteTask` class below
- modify the `count` method above to handle that case, like this:

    ```python
          def count(self, task):
              if task.skipped:
                  self.skipped += 1
              elif task.error:
                  self.failed += 1
              elif task.is_redirect:
                  self.uploaded_redirects += 1
              elif task.is_deletion:
                  self.deleted_files += 1
              else:
                  self.uploaded_files += 1
                  self.uploaded_files_size += task.size

- and then finally use `totals.count(task)` instead of `totals.delete(task)` below

Reply:
done.